DeskKit/DeskKit/View Models/DKContactUsViewModel.m

```diff
@@ -99,6 +103,22 @@ - (NSArray *)sections
     return _sections;
 }
 
+- (NSAttributedString *)attributedTextWithString:(NSString *)string
+{
+    if (string) {
+        return [[NSAttributedString alloc] initWithString:string];
+    }
+    return nil;
+}
+
+- (NSAttributedString *)attributedPlaceholderTextWithString:(NSString *)string
+{
+    if (string) {
+        return [[NSAttributedString alloc] initWithString:string];
+    }
+    return nil;
+}
```

Comment:
Yea, the idea here was in anticipation of being asked in the future to styled these differently. Otherwise I could delete both methods altogether and just use '[[NSAttributedString alloc] initWithString:string]' where they are currently used. Thoughts?